### DISCUSSION TITLE
More information for DbErr::Query

### DISCUSSION THREAD

**Discussion by TheCataliasTNT2k** [SeaQL/sea-orm#1422] [Ideas]
## Motivation

If you get an `DbErr::Query` Error, it would be great, if you could get more information, like 
executed query including parameters
and the database error like "Duplication in column foo while inserting"

This can be useful for other `DbErr` types as well...

## Proposed Solutions

Adding `DbErr::Query::query` and `DbErr::Query::response`

**Comment by billy1624** [comment]
Hey @TheCataliasTNT2k, you can get the database specific error code and error message from it.
https://github.com/SeaQL/sea-orm/blob/f360e79e1e4e7b9b87b2640d26a1ef6114144501/tests/crud/error.rs#L24-L55

**Reply by TheCataliasTNT2k** [reply]
Yes, but I want to get the query and parameters as well.
E.g.: Insert with duplicate column parameter (like username, which is already taken). I would like to know, which parameter was duplicated and which value was the cause for the error.
